### DISCUSSION TITLE
Help with LUT Calibration

### DISCUSSION THREAD

**Discussion by nihal711** [awawa-dev/HyperHDR#264] [Q&A]
I'm currently testing out the latest HyperHDR beta build and its been great so far. I'm using a MS2109 capture card and quite unsure with how I should run the LUT calibration. A mini guide would be much appreciated, thanks! 

Would it auto generate a lut table if I did not run this? Asking as the colours are slightly off and I'm not sure how to go about turning hdr tone mapping on and off automatically (json api allows it but not sure if there's an automation out there)
![lut](https://user-images.githubusercontent.com/28503335/165898907-4fbdc26a-173f-4149-ad38-0c63bcca6e1c.png)

**Comment by awawa-dev** [comment]
Hi
Suggest you to download and install prebuild LUT for MS2109: just follow the instruction to set proper brightness/contrast/saturation for that  grabber (Instruction is on my blog and in the LUT file archive). BTW it's not important for the calibration process but I'm curious: as you can see the screenshot you posted has replaced colors on the gradient (https://ibb.co/515RB4L) but the green color of the button is OK so probably just gradient has shifted angle. Which WWW browser?
There is no automatic HDR detection with an exception of webOS backend developed by the community. But you would need an LG TV.

**Reply by nihal711** [reply]
This was on my desktop using google chrome. Also, an update to my post above. I used the LUT file you provided (along with the bright and contrast/saturation setting) and the results are amazing! 

Using a Samsung TV so I’m guessing that the automation won’t be possible. I’m hoping it will be fine to leave the HDR tone mapping always on? Otherwise I’ll probably just create a dummy switch on my Homebridge instance for turning the HDR on and off via json api.

Thanks once again for the fast response and help!

**Reply by awawa-dev** [reply]
I have an LG, and despite the auto-switch option, I chose to always have HDR enabled on my system and player: SDR content upgraded to HDR looks much better. At least using madVR on Windows gives such a good result.
